src/app/shared/task-provider/task-provider.component.ts

```diff
@@ -11,6 +11,7 @@ export class TaskProviderComponent implements OnInit {
 
     @Input() taskProviderOptions: any = [];
     @Output() initializedTaskProvider = new EventEmitter<FormGroup>();
+    @Output() getTimeProviders = new EventEmitter<string>();
```

Comment:
`onProviderChanged` looks better here, we should not use "time provider" keyword anywhere in the task provider component.
```suggestion
    @Output() onProviderChanged = new EventEmitter<string>();
```

Reply:
done.